Disable generation of documentation

Generation of documentation is not necessary in Buildroot, disable it completely.

Signed-off-by: Dmitry Golubovsky <golubovsky@gmail.com>

Index: linux-pam-1.1.4/doc/Makefile.am
============================================================================
--- linux-pam-1.1.4/doc/Makefile.am	2011-06-21 05:04:56.000000000 -0400
+++ linux-pam-1.1.4/doc/Makefile.am	2012-08-09 05:59:23.000000000 -0400
@@ -2,8 +2,6 @@
 # Copyright (c) 2005, 2006 Thorsten Kukuk <kukuk@suse.de>
 #
 
-SUBDIRS = man specs sag adg mwg
-
 CLEANFILES = *~
 
 dist_html_DATA = index.html
@@ -11,12 +9,4 @@
 #######################################################
 
 releasedocs: all
-	$(mkinstalldirs) $(top_builddir)/Linux-PAM-$(VERSION)/doc/specs
-	cp -av specs/draft-morgan-pam-current.txt \
-		$(top_builddir)/Linux-PAM-$(VERSION)/doc/specs/
-	cp -av $(srcdir)/specs/rfc86.0.txt \
-		$(top_builddir)/Linux-PAM-$(VERSION)/doc/specs/
-	make -C sag releasedocs
-	make -C adg releasedocs
-	make -C mwg releasedocs
-	
+	/bin/true
